### PR TITLE
Miscellaneous changes made when working on Guided Tour

### PR DESCRIPTION
**Pull request by nramabad**
- Rm unnecessary jquery onreadys in experiment files
- Remove extraneous space in snippet feedback
- Remove occasional and random green flicker on page load

### CODE REVIEW

**Review by rsylvian — APPROVED** [review]
Thanks ✅

File: src/js/experiments/surfaceFullExampleConfig.js [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test',
-    experimentContainer: '#full-config-example',
-  }).then((variation) => {
-    if (variation === 'treatment') {
-      $('#full-config-example').css('visibility', 'visible');
-      $('#full-config-example').click(function () {
-        window.AnalyticsClient.trackAction('docs-full-config-example-link', {
-          link: this.id,
-        });
+// https://app.optimizely.com/v2/projects/16812830475/experiments/20742690126/variations
```
Comment (teesloane) [inline src/js/experiments/surfaceFullExampleConfig.js:1]: I think the removal of this just got merged.

**Review by teesloane — APPROVED** [review]
LGTM

File: src/js/services/progressbar.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const init = () => {
+  document.body.style.backgroundColor = '#fff';
```
Comment (chantelle11) [inline src/js/services/progressbar.js:2]: Why was not a class added with background-color: #fff?

**Review by chantelle11 — APPROVED** [review]
Great work on this! 🥳🤩🌺

File: src/js/services/progressbar.js (hunk 2 above)
Comment (nramabad) [reply src/js/services/progressbar.js:2]: styles apply more strongly in css hierarchy.

**Pull request merged**